tests/backend/mongodb/test_connection.py

```diff
@@ -14,3 +18,21 @@ def test_get_connection_returns_the_correct_instance():
     conn = connect(**config)
     assert isinstance(conn, Connection)
     assert isinstance(conn, MongoDBConnection)
+
+
+@mock.patch('pymongo.MongoClient.__init__')
+@mock.patch('time.sleep')
+def test_connection_error(mock_sleep, mock_client):
+    from pymongo.errors import ConnectionFailure
```

Comment:
for code other than bigchaindb we can import at the following the PEP 8 style guide